### ISSUE TITLE
Readme shows old method of using pip

### ISSUE THREAD

**Issue by another4til7** [0xabu/pdfannots#88]
**Trying to follow the instructions to install using pip fails:**

> pip3 install pdfannots
bash: /home/[user]/.local/bin/pip3: /usr/bin/python: bad interpreter: No such file or directory

A stack overflow article about basically the same problem (https://stackoverflow.com/questions/51373063/pip3-bad-interpreter-no-such-file-or-directory#51373253) notes: 

    You've got a whole slew of different Python installations, plus at least one former Python installation that you deleted.

    Situations like this are exactly why running pip or pip3 directly is no longer recommended, in favor of:

    python3 -m pip install whatever

When I try that, pdfannots now installs
> python3 -m pip install pdfannots
Defaulting to user installation because...
...
Installing collected packages: cryptography, charset-normalizer, pdfminer.six, pdfannots
Successfully installed charset-normalizer-3.2.0 cryptography-41.0.3 pdfannots-0.4 pdfminer.six-20221105

**to recap / tl;dr**
Solution: Update install instructions to match current pip practice,

Change README.md line 41 from:
pip3 install pdfannots

to:
python3 -m pip install pdfannots

**Comment by another4til7** [comment]
to recap / tl;dr

In order to install per current pip practice, change line 41 
from:
pip3 install pdfannots

to:
python3 -m pip install pdfannots

**Comment by 0xabu** [comment]
Thanks for the suggestion.
